src/app/components/collectibles/src20-image.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+import { css } from 'leather-styles/css';
+
+import { Src20AvatarIcon } from '@leather.io/ui';
+
+interface Src20ImageProps {
+  alt?: string;
+  src: string;
+}
+export function Src20Image(props: Src20ImageProps) {
+  const { alt, src } = props;
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [width, setWidth] = useState(0);
+
+  if (isError) return <Src20AvatarIcon width="xl" />;
```

Comment:
I don't think you want to resize the icon here?

Reply:
@camerow I would move this file out of `collectibles` bc being used in the asset list?

Reply:
Yeah it seems to make no difference if I resize it or not (perhaps that's the default size). 

I can move it, should I move to a new `/components/src20/src20-image.tsx` or just put it in the `src20-tokens-asset-list`?

Reply:
Updated the location, let me know if it looks ok to you.